.github/workflows/publish.yml

```diff
@@ -85,6 +111,23 @@ jobs:
           chartpress --no-build
           ./tools/set-chart-yaml-annotations.py
 
+      - name: Publish images and chart with chartpress
+        if: steps.publishing.outputs.publishing
+        env:
+          GITHUB_REPOSITORY: "${{ github.repository }}"
+        run: |
           # Package the Helm chart and publish it to the gh-pages branch of
           # the jupyterhub/helm-chart repo.
           ./ci/publish
+
+      - name: Package helm chart as a CI artifact
+        if: steps.publishing.outputs.publishing == ''
+        run: helm package jupyterhub
```

Comment:
Question here: this runs `helm package` and uses the resulting tgz as the artifact, which then gets re-archived as a .zip (actions doesn't support unmodified single-file artifacts, they are *always* zipped up).

Should we just make the `jupyterhub` directory itself the artifact without running `helm package`? I see that the result of `helm package` does rewrite Chart.yaml, but it appears to only be re-serializing the yaml (e.g. without comments) without any actual changes.

Reply:
My initial thoughts are that the closer the artifact is to the final published artifact the better, for example to cover the case if `helm package ...` has some weird behaviour, or does something different in future.